docs/content/concepts/tokenomics/gas-in-sui.mdx

```diff
@@ -21,7 +21,49 @@ _The Gas Fees section for a transaction block displayed on a Sui network explore
 
 ## Gas prices {#gas-prices}
 
-The [reference gas price](gas-pricing.mdx#computation-gas-prices) translates the real-time cost of executing a transaction into SUI units and the validator set updates it at each epoch boundary. Similarly, the [storage price](gas-pricing.mdx#storage-gas-prices) translates the long-term cost of storing data on chain into SUI units and updates infrequently; often remaining constant for various consecutive epochs. During regular network operations, all Sui users can expect to pay the reference gas price and storage price for computation and storage, respectively.
+The Sui gas-pricing mechanism achieves three outcomes: delivering low, predictable transaction fees, incentivizing the network's operators (validators) to optimize their transaction processing operations, and preventing denial of service attacks.
+
+This enables you to focus on using the Sui network to provide the best user experience without needing to forecast the current market price of gas fees. Since validators agree on a network-wide reference price at the start of each epoch, you can use the reference price as a credible anchor when submitting transactions. Moreover, the price setting mechanism rewards good validator behavior, thus aligning incentives between SUI token holders, validators, and Sui users.
```

Comment:
```suggestion
This pricing mechanism enables you to focus on using the Sui network to provide the best user experience without needing to forecast the current market price of gas fees. Because validators agree on a network-wide reference price at the start of each epoch, you can use the reference price as a credible anchor when submitting transactions. Moreover, the price setting mechanism rewards good validator behavior, thus aligning incentives between SUI token holders, validators, and users.
```
Not sure if the phrase I used is best, but feels weird using "this" at the start of a new para. Admittedly, the fact that the previous para is the first and only one sentence makes that thought debatable.